packages/core/testing/src/test_bed_common.ts

```diff
@@ -55,6 +58,13 @@ export type TestModuleMetadata = {
    * @see https://angular.io/errors/NG8001 for the description of the problem and how to fix it
    */
   errorOnUnknownElements?: boolean;
+  /**
+   * Whether errors should be thrown on unknown properties.
+   * Defaults to `false`, where the error is simply logged.
+   * If sets to `true`, the error is thrown.
```

Comment:
Just a minor update to the docs (similar updates can also be applied to the `errorOnUnknownElements` docs above):
```suggestion
   * Whether errors should be thrown when unknown properties are present in component's template.
   * Defaults to `false`, where the error is logged into a console.
   * If set to `true`, the error is thrown.
```

Reply:
... and also similar updates can be made to the docs below.